Calculator/ViewController/ViewController.swift

```diff
@@ -28,54 +80,172 @@ class ViewController: UIViewController {
                 lblTextLabel.text = lblTextLabel.text! + "0"
```

Comment:
```suggestion
                lblTextLabel.text = lblTextLabel.text + "0"
```
tương tự với các cái bên dưới